validphys2/src/validphys/tests/test_regressions.py

```diff
@@ -69,15 +69,19 @@ def test_expcovmat(data_config):
 
 @make_table_comp(parse_exp_mat)
 def test_t0covmat(data_witht0_config):
-    return API.groups_covmat_no_table(**data_witht0_config)
+    d = dict(data_witht0_config)
+    d['use_cuts'] = 'internal'
```

Comment:
You can use this instead: https://github.com/NNPDF/nnpdf/blob/eeae19b15c960b4f8f974c3ac2ba2af36ed845bc/validphys2/src/validphys/tests/conftest.py#L94

Reply:
Wonderful, good spot.